bin/cli.js

```diff
@@ -260,8 +262,8 @@ const onExit = code => {
 };
 
 const config = getConfig();
-console.log('Metatest CLI Stub');
-console.log(`Metatests final config:\n${JSON.stringify(config, null, 2)}\n`);
+if (config.verbosity >= 3)
```

Comment:
Also, I don't think we use multiline if-without-brackets style anywhere. It's either one-line or with brackets.